prometheus/testutil/promlint/promlint.go

```diff
@@ -328,8 +329,7 @@ var (
 		"days":    "seconds",
 		"weeks":   "seconds",
 		// Temperature.
-		"kelvin":     "celsius",
-		"kelvins":    "celsius",
+		"kelvins":    "kelvin",
 		"fahrenheit": "celsius",
 		"rankine":    "celsius",
```

Comment:
Just a random thought, should rankine be suggested as kelvin as they are both absolute zero based?

Reply:
That makes sense, but I'd be careful here. We don't really want to push people towards Kelvin. It's only that there are cases where you really cannot _not_ use Kelvin, in which case the lint warnings are really annoying, given there's no "lint-ignore" pragma possible in the Prometheus metrics format.

If you get the lint warning "Use `celsius` instead of `rankine`", and you have strong reasons for using absolute temperature, you are probably deep enough into it that you'll recognize that switching to `kelvin` will still fix the warning.